example/usage_quick.ipynb

```diff
@@ -142,6 +142,22 @@
     "_ = snl.records_diff(variables=[\"Confirmed\"], window=7)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`Scenario.show_complement()` method is useful to show the kinds of complement. The details of complement are explained in [Usage: datasets](https://lisphilar.github.io/covid19-sir/usage_dataset.html) section."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "complement_snl_df = snl.show_complement()"
```

Comment:
The dataframe seems not shown. Please remove `complement_snl_df = `from this line.

Reply:
Added `complement_snl_df.tail()`.